compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -1801,12 +1801,20 @@ extern "C" {
     ///
     /// In order for Rust-C LTO to work, module flags must be compatible with Clang. What
     /// "compatible" means depends on the merge behaviors involved.
-    pub fn LLVMRustAddModuleFlag(
+    pub fn LLVMRustAddModuleFlagU32(
```

Comment:
maybe we can keep its name so that the change can be smaller; consider only riscv needs to set a string flag.

Reply:
I think it doesn't hurt because the current naming scheme is more consistent(method name with a type suffix. And we don't have to care about patch size.